packages/server/src/drizzle/schema.ts

```
@@ -23,7 +28,7 @@ export type SelectAuth = typeof authTable.$inferSelect;
 
 export const clientTable = sqliteTable("client", {
 	id: text("id", { length: 128 })
-		.$defaultFn(() => `CL_` + createId())
+		.$defaultFn(() => genEntityId(`CL`))
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)